class/defaults.yml

```diff
@@ -40,7 +40,7 @@ parameters:
     namespace: syn-${_instance}
     name: keycloak
     charts:
-      keycloakx: "1.3.2"
+      keycloakx: "1.5.0"
```

Comment:
What about #158 ? This would update Keycloak to v19. Any reason why not going v19 yet?

Reply:
Yes, because v19 introduces a new UI. Keycloak itself is in a heavy change, so just do one major upgrade reduces the risk of not understanding the changes.